core/src/main/java/ai/timefold/solver/core/config/solver/SolverConfig.java

```diff
@@ -280,6 +290,22 @@ public SolverConfig(@NonNull SolverConfig inheritedConfig) {
         inherit(inheritedConfig);
     }
 
+    /**
+     * For testing purposes only.
```

Comment:
I would not say it is solely used for tests, as we pass it to the solver factory.

Reply:
The solver factory gets the default value, unless the config says something.
If a test needs to override it, it should use the config. Therefore I'd say that the solver config really is _only_ for testing.